### ISSUE TITLE
第十一章 情感分类实战源码 sentiment_analysis_cell.py 编译出错

### ISSUE THREAD

**Issue by BruceWeiii** [dragen1860/Deep-Learning-with-TensorFlow-book#128]
环境 python3.7，tensorflow2.0.0，未使用gpu
错误语句：line 96,   model.fit(db_train, epochs=epochs, validation_data=db_test)
错误描述：tensorflow.python.eager.core._SymbolicException: Inputs to eager execution function cannot be Keras symbolic tensors, but found [<tf.Tensor 'my_rnn/simple_rnn_cell/cond/Identity:0' shape=(None, 100) dtype=float32>, <tf.Tensor 'my_rnn/simple_rnn_cell_1/cond/Identity:0' shape=(128, 64) dtype=float32>]

其他：LSTM实战源码出现同样错误

**Comment by sfw134** [comment]
看不懂报错，感觉没啥问题，坐等回答

**Comment by YuanchenHua** [comment]
感觉是model.fit()函数调用方式,input输入的格式不对
不怕麻烦手写主循环应该就可以了(我没写过不知道,但应该可以哈哈哈啊哈哈).
或者改一下input的格式,改成tensor而不是keras_symbolic_tensors(具体咋改我也不知道,哈哈哈哈哈哈)

**Comment by cymx66688** [comment]
我也遇到了同样的问题

**Comment by 2226171237** [comment]
把SimpleRNNCell的dropout去掉就好了，不知道为什么。。。

**Comment by cymx66688** [comment]
> 去掉就好了，不知道为什

model.compile(optimizer = optimizers.RMSprop(0.001),
                  loss = losses.BinaryCrossentropy(),
                  metrics=['accuracy'], run_eagerly=True)
在compile中加run_eagerly=True,就可以了

**Comment by 2226171237** [comment]
> > 去掉就好了，不知道为什
> 
> model.compile(optimizer = optimizers.RMSprop(0.001),
> loss = losses.BinaryCrossentropy(),
> metrics=['accuracy'], run_eagerly=True)
> 在compile中加run_eagerly=True,就可以了

感谢

**Comment by faultXXXXL** [comment]
在model.compile里面加上experimental_run_tf_function=False就能跑起来

**Comment by WzyPaopao** [comment]
> 在model.compile里面加上experimental_run_tf_function=False就能跑起来

我这样过后报了一个关于cuda的错误
